Submitted By:            BLFS Book <blfs-book@linuxfromscratch.org>
Date:                    2004-11-19
Initial Package Version: 2.8.2
Upstream Status:         Not submitted
Origin:                  PLD site; sent to LFS-Dev by John Lawrence Young
             See: http://archives.linuxfromscratch.org/mail-archives/blfs-dev/2004-August/007318.html
Description:             Fixes build problem due to API change in libexif >= 0.6.9

$LastChangedBy$
$Date$

diff -Naur nautilus-2.8.2-orig/components/image_properties/nautilus-image-properties-view.c 
           nautilus-2.8.2/components/image_properties/nautilus-image-properties-view.c
--- nautilus-2.8.2-orig/components/image_properties/nautilus-image-properties-view.c	2004-03-09 11:23:37.000000000 +0000
+++ nautilus-2.8.2/components/image_properties/nautilus-image-properties-view.c	2004-11-19 23:12:05.726408904 +0000
@@ -125,12 +125,14 @@
 {
 	struct ExifAttribute *attribute;
 
+	char buf[1024];
+
 	attribute = (struct ExifAttribute *)data;
 	if (attribute->found) {
 		return;
 	}
 
-        attribute->value = g_strdup (exif_content_get_value (content, attribute->tag));
+        attribute->value = g_strdup (exif_content_get_value (content, attribute->tag, buf, sizeof(buf)));
 	if (attribute->value != NULL) {
 		attribute->found = TRUE;
 	}
